code/modules/jobs/job_types/research_director.dm

```diff
@@ -39,9 +39,17 @@
 	changed_maps = list("OmegaStation")
 
 	mail_goodies = list(
-		/obj/item/storage/box/monkeycubes = 30,
+		/obj/effect/spawner/lootdrop/aimodule_neutral = 15,
+		/obj/item/transfer_valve = 15,
+		/obj/effect/spawner/lootdrop/aimodule_harmless = 10,
+		/obj/item/clothing/mask/facehugger/toy = 5,
 		///obj/item/circuitboard/machine/sleeper/party = 3,
-		/obj/item/borg/upgrade/ai = 2
+		/obj/item/borg/upgrade/ai = 2,
+		/obj/effect/spawner/lootdrop/surgery_tool_alien = 2,
+		/obj/effect/spawner/lootdrop/engineering_tool_alien = 2,
+		/obj/effect/spawner/lootdrop/organ_spawner = 2,
+		/obj/effect/spawner/lootdrop/aimodule_harmful = 1,
```

Comment:
these could be a problem

Reply:
Well there's already a couple harmful lawset boards in AI upload anyway for the RD and those never caused any issues. I doubt it ever find any real use tbh but it's cool flavor.

Reply:
fair enough, if it ends up being a problem they can be removed